cells/klayout/pymacros/cells/draw_mos.py

```diff
@@ -646,7 +646,7 @@ def draw_pmos(layout, l, w, ld, nf, grw, bulk, volt, deepnwell, pcmpgr):
 
         # Inserting Tie
         if (w * ld) < min_cmp_area:
-            tie_violat = (min_cmp_area/ld - w)/2 * tol
+            tie_violat = (min_cmp_area * tol/ld - w)/2
```

Comment:
Order of operations is confusing. Please add parans around the prefered order of operations